src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx

```diff
@@ -91,12 +92,17 @@ const CheckInConfirmation = props => {
               data-testid="travel-pay-message"
             >
               <div>
-                <strong>{t('reimbursement-claim-number')}</strong>
-                <br />
-                {travelPayClaimData.claimId}
+                {t('check-travel-claim-status')}
                 <br />
```

Comment:
Could we wrap things in paragraph tags instead?

Reply:
yes that is a better idea! I got caught up in using what was already there. Great catch